### ISSUE TITLE
The code will not be executed

### ISSUE THREAD

**Issue by warisamin25** [AakshiKochhar/TrantorInc#1]
https://github.com/AakshiKochhar/TrantorInc/blob/0811f46096b450d9e7d2cbb6ab40664f77323832/webapi.py#L37
Can you explain the function.

**Comment by AakshiKochhar** [comment]
Hi, the point of line 37 till line 41 was to help determine in the login function whether one called a GET request or a POST request. In order to check whether one reaches the page through a GET request or POST request, I implemented the if/else statement to see which page I would reach if I got a POST request. In order to test this I had created another function called "user". This function would pass the user's name as an argument, reload the website, and display the user's name on the screen. So for line 37, _if request.method == "POST"_, then the point was to get the information from the textbox, store it through request.form and use that to send the user to the user page (which doesn't exist anymore), where one could display the user's name.

However, after finishing up the testing, I removed the "user" function but didn't remove the associated code (line 37-41). Line 37 through 41 doesn't add much value to the overall code without that function, and although I know that the whole point of a GET request is to use it when one wants to view something -- without changing any data -- a POST request is used for the opposite (used to insert or update remote data). 

Since the user would not be inserting or updating any remote data in this game, a POST request wouldn't be necessary

**Comment by warisamin25** [comment]
Will this line ever execute?

**Comment by AakshiKochhar** [comment]
No, not without the "user" function. All data that gets collected are through GET requests only. I could remove line 37-42 or add the "user" function to show the implementation of the POST request
